src/app/FakeLib/WiXHelper.fs

```diff
@@ -77,6 +78,24 @@ let rec wixDir fileFilter asSubDir (directoryInfo : DirectoryInfo) =
             directoryInfo.Name dirs compo
     else sprintf "%s%s" dirs compo
 
+/// Retrieves the file id of the first file in WiXString, which name matches fileRegex
+/// ## Parameters
+///  - `wiXString` - The directory string which was generated by wixDir
+///  - `fileRegex` - Regex which matches the file name
+///
+/// ## Sample
+/// let directoryString = wixDir (fun file -> true) true (DirectoryInfo directoryWithFilesForSetup)
```

Comment:
I think the sample code needs to be indented like https://github.com/fsharp/FAKE/blob/master/src/app/FakeLib/ZipHelper.fs#L165